cfgov/v1/models/browse_filterable_page.py

```diff
@@ -68,7 +72,19 @@ class NewsroomLandingPage(BrowseFilterablePage):
 
     objects = PageManager()
 
-    def base_query(self, hostname):
-        """ Modify the base query for Newsroom to exclude research reports """
-        reports = [x[0] for x in dict(ref.categories)['Research Report']]
-        return AbstractFilterPage.objects.live_shared(hostname).exclude(categories__name__in=reports)
+    @classmethod
+    def eligible_categories(cls):
+        categories = dict(ref.categories)
+        return sorted(itertools.chain(*(
+            dict(categories[category]).keys()
+            for category in ('Blog', 'Newsroom')
+        )))
```

Comment:
Don't these categories ('Blog' and 'Newsroom') need to get mapped to their subcategories as is done in `clean_categories` here? https://github.com/cfpb/cfgov-refresh/blob/7c6e15e1eaa5fa61d4e46d0163f5595e21b6c609/cfgov/v1/forms.py#L132